Realm/ObjectServerTests/RLMObjectServerTests.mm

```diff
@@ -515,7 +513,7 @@ - (void)testInvalidPasswordAuthentication {
 
     XCTestExpectation *expectation = [self expectationWithDescription:@"login should fail"];
 
-    [self.app loginWithCredential:credentials completion:^(RLMUser * user, NSError * error) {
+    [self.app loginWithCredential:credentials completion:^(RLMUser * user, NSError *error) {
```

Comment:
There are a bunch of extra spaces in this file before `*`, but they were already there